src/Watcher.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using k8s.Exceptions;
+using Microsoft.Rest;
+using Microsoft.Rest.Serialization;
+
+namespace k8s
+{
+    public enum WatchEventType
+    {
+        [EnumMember(Value = "ADDED")] Added,
+

```

Comment:
I'm definitely not a csharp style expert, but it feels weird to have newlines between these values.

Reply:
I really love `go fmt`

we might introduce something like https://github.com/dotnet/codeformatter and contribution guide. I will create another issue to track this.